websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,108 @@
+			presenceData.details = "Viewing the FAQ page";
+		else if (pathname.startsWith("/contact"))
+			presenceData.details = "Viewing the contact page";
+		else if (pathname.startsWith("/terms"))
+			presenceData.details = "Viewing the terms page";
+		else if (pathname.startsWith("/privacy"))
+			presenceData.details = "Viewing the privacy page";
+		else if (pathname.startsWith("/legal"))
+			presenceData.details = "Viewing the legal page";
+		else if (pathname.startsWith("/forgot-password"))
+			presenceData.details = "Resetting their password";
+	} else if (hostname.startsWith("my")) {
+		if (pathname === "/") {
+			if (
+				document.querySelector(
+					"#root > div > div.sc-gKsewC.sc-iBPRYJ.sc-gGiJkG.efCouF.cOWINi.jzdkxv"
+				)
+			)
+				presenceData.details = "Logging in";
+			else if (
+				document.querySelector("#root > div > div > div > div.sc-jsNsAp.byZiWf")
+			)
+				presenceData.details = "Choosing mode";
+		} else if (pathname.startsWith("/signup"))
+			presenceData.details = "Signing up";
+		else if (pathname.startsWith("/welcome"))
+			presenceData.details = "Getting started";
+		else if (["/focus", "/relax", "/sleep"].includes(pathname)) {
+			switch (pathname) {
+				case "/focus":
+					presenceData.state = `Focusing • ${document
+						.querySelector(
+							"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+						)
+						.textContent.trim()}`;
+					break;
+				case "/relax":
+					presenceData.state = `Relaxing • ${document
+						.querySelector(
+							"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+						)
+						.textContent.trim()}`;
+					break;
+				case "/sleep":
+					presenceData.state = `Sleeping • ${document
+						.querySelector(
+							"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+						)
+						.textContent.trim()}`;
+					break;
+			}
+
+			const songTypeInfo = document
+				.querySelector(
+					"#root > div > div > div.sc-VhGJa.kMMYFW > div.sc-jwBoPJ.glJHPW > div > div.sc-chbBtj.ciMhIb > div > div > div > div > div.sc-leFDRL.iaABgi"
+				)
+				.textContent.trim()
+				.split(" • ");
+
+			presenceData.details = `${document
+				.querySelector(
+					"#root > div > div > div.sc-VhGJa.kMMYFW > div.sc-jwBoPJ.glJHPW > div > div.sc-chbBtj.ciMhIb > div > div > div > div > div.sc-hDjjHo.jwdsSo"
+				)
+				.textContent.trim()} • ${songTypeInfo[0]}`;
+
+			presenceData.largeImageKey = document
+				.querySelector(
+					"#root > div > div > div.sc-VhGJa.kMMYFW > div.sc-jwBoPJ.glJHPW > div > div.sc-chbBtj.ciMhIb > div > div > div > img"
+				)
+				.getAttribute("src");
+			presenceData.smallImageKey = "logo";
+			presenceData.smallImageText = songTypeInfo[1];
+		} else if (pathname.startsWith("/user"))
+			presenceData.details = "Viewing user information";
```

Comment:
which user?

Reply:
Would you like it to say this?

Viewing **my** user information